web/src/admin/admin-overview/cards/RecentEventsCard.ts

```diff
@@ -1,7 +1,8 @@
 import "@goauthentik/admin/events/EventInfo";
-import { ActionToLabel, EventGeo } from "@goauthentik/admin/events/utils";
+import { EventGeo } from "@goauthentik/admin/events/utils";
 import { DEFAULT_CONFIG } from "@goauthentik/common/api/config";
 import { EventWithContext } from "@goauthentik/common/events";
+import { actionToLabel } from "@goauthentik/common/labels";
```

Comment:
_All_ of the changes are of this: referring to the function in a different location, and changing the casing to indicate it's a _function_ rather than a type or class.